STUDIO/org.openl.rules.project/src/org/openl/rules/project/model/OpenAPI.java

```diff
@@ -4,8 +4,18 @@
 
 public final class OpenAPI {
     public enum Mode {
-        RECONCILIATION,
-        GENERATION
+        RECONCILIATION("Reconciliation"),
+        GENERATION("Tables generation");

```

Comment:
UI messages should be declared inside HTML or in localization properties

Reply:
done